### PR TITLE
Update CoreOS AMI

### PR DESCRIPTION
**Pull request by mazzy89**
Update CoreOS AMI

### CODE REVIEW

**Comment by mazzy89** [comment]
@calvix should I add a note in the changelog here https://github.com/giantswarm/aws-operator/blob/f96a5b9a30c85d9184f65bbaa54e6ae381e5908f/service/controller/v24/version_bundle.go#L9?

**Comment by calvix** [comment]
Yep message that coreos got updated should be there

File: service/controller/v24/version_bundle.go [hunk 1]
```diff
@@ -12,6 +12,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Mount /var/log directory in an EBS Volume.",
 				Kind:        versionbundle.KindAdded,
 			},
+			{
+				Component:   "containerlinux",
+				Description: "Update CoreOS to 2023.4.0.",
```
Comment (teemow) [inline service/controller/v24/version_bundle.go]:
```suggestion
				Description: "Update CoreOS to 2023.4.0. Fixes CVE-2019-8912",
```

**Comment by mazzy89** [comment]
2 e2e fails with the following error:

```
integration/test/draining/draining_test.go:26:2: cannot find package "github.com/giantswarm/aws-operator/service/controller/v21/key" in any of:
	/go/src/github.com/giantswarm/aws-operator/vendor/github.com/giantswarm/aws-operator/service/controller/v21/key (vendor tree)
	/usr/local/go/src/github.com/giantswarm/aws-operator/service/controller/v21/key (from $GOROOT)
	/go/src/github.com/giantswarm/aws-operator/service/controller/v21/key (from $GOPATH)
FAIL	github.com/giantswarm/aws-operator/integration/test/draining [setup failed]
```

this is due to the fact they look for the bundle `v21` which has been removed in a recent cleanup. Please @xh3b4sd could you point out there to which bundle should I update the integration test? The one there is `v21`. Is it ok the last one that is `v24`?

**Comment by xh3b4sd** [comment]
I fixed that in my current PR. Missed it when dropping the bundles. Ignore it here. I merge my PR later.

**Pull request merged**